Storage/src/SigningHelper.php

```diff
@@ -181,9 +184,12 @@ public function v2Sign(ConnectionInterface $connection, $expires, $resource, $ge
 
         $stringToSign = $this->createV2CanonicalRequest($toSign);
 
-        $signature = $credentials->signBlob($stringToSign, [
-            'forceOpenssl' => $options['forceOpenssl']
-        ]);
+        // Use exponential backOff
+        $signature = $this->retrySignBlob(function () use ($credentials, $stringToSign, $options) {
+            return $credentials->signBlob($stringToSign, [
+                'forceOpenssl' => $options['forceOpenssl']
+            ]);
+        });
```

Comment:
nit, could be cleaner using arrow functions:

```suggestion
        $signature = $this->retrySignBlob(fn () => $credentials->signBlob($stringToSign, [
            'forceOpenssl' => $options['forceOpenssl']
        ]));
```

Reply:
Thank you for your suggestion regarding the use of arrow functions. I have reviewed the code and applied the changes as recommended. It does indeed make the code cleaner and more concise.